src/main/java/com/spotinst/sdkjava/model/api/ocean/kubernetes/ApiK8sVngResourceLimits.java

```diff
@@ -0,0 +1,45 @@
+package com.spotinst.sdkjava.model.api.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiK8sVngResourceLimits implements IPartialUpdateEntity {
+    @JsonIgnore
+    private Set<String>                isSet;
+    private Integer                    maxInstanceCount;
```

Comment:
Another parameter minInstanceCount should be there as per documentation.

Reply:
The API show in postmane only has the following:
"resourceLimits": {
            "maxInstanceCount": 5
        },